doc/downgrade.8.md

```diff
@@ -92,18 +106,14 @@ As per the usage syntax, any options supplied after the **\--** character sequen
 
 ## DEFAULT BEHAVIORS
 
-By default, **downgrade** will search both local caches and the ALA.
+By default, **downgrade** will search both local caches and the ALA. 
```

Comment:
:art: you introduced some trailing whitespace here.

Reply:
Woops, fixed.